deeplinkdispatch-base/src/main/java/com/airbnb/deeplinkdispatch/base/Utils.kt

```diff
@@ -20,4 +24,42 @@ object Utils {
     private fun getBytes(inputStream: InputStream): ByteArray {
         return inputStream.readBytes()
     }
+
+    @JvmStatic
+    fun isComponentTypeRoot(nodeMetadata: Byte): Boolean =
```

Comment:
Would these be better colocated with the definition of the enum, instead of in a utils class?